deploy/environments/applications.yaml

```diff
@@ -138,6 +140,7 @@ spack:
     - py-morph-tool@2.8.0%gcc ^neuron%intel
     - py-neurodamus+all_deps
     - py-notebook
+    - py-opencv-python
```

Comment:
Do we want to build this explicitly? It should already be in the dependency tree for `py-atldld`

Reply:
It's probably enough to have it as a dependency of `py-atldld` for now. Is it just a matter of removing this line, or is there something else I have to do?

Reply:
Just remove the line should be good enough.

Reply:
```suggestion
```
This and we're good to merge.

Reply:
OK, fixed in 39c7d09af5

Reply:
And having two entries for `py-atldld` in `applications.yaml` is correct? I'm not so sure what each of them is for.